src/service/poc_mobile.proto

```diff
@@ -438,6 +438,89 @@ message radio_reward {
   repeated boosted_hex boosted_hexes = 10;
 }
 
+message radio_reward_v2 {
+  enum boosted_hex_status {
+    boosted_hex_status_eligible = 0;
+    boosted_hex_status_location_score_below_threshold = 1;
+    boosted_hex_status_radio_threshold_not_met = 2;
+    boosted_hex_status_service_provider_ban = 3;
+  }
+
+  message location_trust_score {
+    uint64 meters_to_asserted = 1;
+    // value is 0.0 to 1.0 multiplied by 1000
+    uint16 trust_score = 2;
+  }
+
+  message covered_hex {
+    uint64 location = 1;
+    // modeled coverage points * rank_multiplier * assignment_multiplier
+    uint64 base_coverage_points = 2;
+    // base_coverage_points * (boosted_multiplier - 1)
+    uint64 boosted_coverage_points = 3;
+
+    oracle_boosting_assignment urbanized = 4;
+    oracle_boosting_assignment footfall = 5;
+    oracle_boosting_assignment landtype = 6;
```

Comment:
not a huge benefit, but the only benefit of keeping them under a sub-field I can think of is ability to have the field identifiers (those 4, 5, 6) sequential in case we add a new `oracle_boosting_assignment` field in the future

Reply:
I believe assignments should be separated as now, to have comprehensive information about forming an `assignment_multiplier`